.stylelintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "rules": {
```

Comment:
Let us add `stylelint-config-standard` and extend it here.

Reply:
ok. doing that right away